src/webui/features/ExploreComics.tsx

```diff
@@ -44,15 +44,13 @@ export default function Feature()
 
     return <Column gap='small' alignX='stretch'>
         <OrderRow selected='popular' orderChangeHandler={handleOrderChange} />
-        {
-            posts !== undefined
-                ? <PostPanelList
-                    posts={posts}
-                    followHandler={handleFollow}
-                    rateHandler={handleRate}
-                    detailsHandler={handleDetails}
-                />
-                : <Loading />
-        }
+        <LoadingContainer data={posts}>
+            <PostPanelList
```

Comment:
After the merge of #185, the profileHandler is a required input for this component